Src/FluentAssertions/Collections/GenericCollectionAssertions.cs

```diff
@@ -813,6 +813,8 @@ public AndConstraint<TAssertions> Contain(IEnumerable<T> expected, string becaus
     /// Asserts that a collection of objects contains at least one object equivalent to another object.
     /// </summary>
     /// <remarks>
+    /// <b>Note:</b> You cannot use this method to assert whether a collection contains a subset of another collection.
+    /// Use: e.g. <see cref="BeSubsetOf"/> with the complete collection as expectation.
```

Comment:
The remarks are not shown when scrolling in intellisense suggestions in VS.
So I think we should improve the summary instead.
And reading the summary now I can see why consumers can get confused by it.

![image](https://github.com/fluentassertions/fluentassertions/assets/919634/cb707b92-3b6b-4b0c-a415-84588b2ee3a0)

Reply:
That's a Visual Studio limitation. This is what I see in Rider:

![image](https://github.com/fluentassertions/fluentassertions/assets/572734/9df05bea-ba9c-4b25-af18-a14e54503dee)

Reply:
I see that as well in VS when hovering over the method definition.

Reply:
Here's an improved version:

```csharp
/// <summary>
    /// Asserts that at least one object in the collection is equivalent to <paramref name="expectation"/>.
    /// </summary>
    /// <remarks>
    /// <para>
    /// By default, objects within the collection are seen as equivalent to the expected object when both object graphs have equally named properties with the same
    /// value, irrespective of the type of those objects.
    /// Notice that actual behavior is determined by the global defaults managed by <see cref="AssertionOptions"/>.
    /// </para>
    /// <para>
    /// <b>Important:</b> You cannot use this method to assert whether a collection contains a subset of another collection.
    /// Use e.g. <see cref="BeSubsetOf"/> instead.
    /// </para>
    /// </remarks>
```

![image](https://github.com/fluentassertions/fluentassertions/assets/572734/cb40b57a-157b-4b58-ad08-f361f93241a2)

Reply:
Can we use `<em>` instead of `<b>`?

Reply:
> That's a Visual Studio limitation. This is what I see in Rider:

VS also renders remarks etc, when hovering the method definition.

![image](https://github.com/fluentassertions/fluentassertions/assets/919634/707dcf2f-15fa-47da-957e-bf8b6ac7cf32)

Reply:
> Can we use `<em>` instead of `<b>`?

Sure... but why?

Reply:
Well, I guess it should be `<strong>`, but because you make it bold for a reason, so your probably want to convey a message. In this case the urgency that it is important?

https://www.w3docs.com/snippets/html/what-is-the-difference-between-b-and-strong-i-and-em-tags.html

Reply:
Thanks for sharing this link 👍🏻

See clear now 🙃

Reply:
Hmm I don't like how this looks in Rider.. I will stick with `<b>` only